WalletWasabi.Fluent/ViewModels/CoinControl/Core/ItemBase.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections.Generic;
+using NBitcoin;
+using WalletWasabi.Blockchain.Analysis.Clustering;
+using WalletWasabi.Fluent.Helpers;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinControl.Core;
+
+public abstract class ItemBase
```

Comment:
The naming is really confusing, can we keep naming classes as in other parts and avoid generic names like item.

Reply:
The ItemBase is not really generic item, it has coin specific properties.

Reply:
SmartCoin and Pocket don't share same base model class, we shouldn't mix two here either
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Fluent/Models/Pocket.cs
https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/Blockchain/TransactionOutputs/SmartCoin.cs#L15

Reply:
![image](https://user-images.githubusercontent.com/2297442/201648546-ca9bcb1d-ad07-4b83-bc4c-fd4382350537.png)

Reply:
Fixed.